Resources/docs/index.rst

```diff
@@ -0,0 +1,378 @@
+Connector installation
+======================
+
+Add ConnectionsBundle and OXIDConnectorBundle to ``AppKernel.php``
+
+.. code-block:: php
+
+    new ONGR\ConnectionsBundle\ONGRConnectionsBundle(),
+    new ONGR\OXIDConnectorBundle\ONGROXIDConnectorBundle(),
+
+..
+
+Then in your bundle you will need to extend ``Documents`` and ``Entities`` provided in OXIDConnectorBundle
```

Comment:
If i remember correctly extending ``Entity`` is optional, but you HAVE to extend ``Documents`` because they are abstract.